vars/main.yml

```diff
@@ -18,6 +18,8 @@ patroni_superuser_username: "postgres"
 patroni_superuser_password: "postgres-pass"  # please change password
 patroni_replication_username: "replicator"
 patroni_replication_password: "replicator-pass"  # please change password
+patroni_restapi_username: "patroni"
+patroni_restapi_password: "restapi-pass"  # please change password
```

Comment:
I suggest placing these variables under the `patroni_restapi_port` variable to group the restapi variables.